tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,163 @@
+"""
+The purpose of this test case is to delete at least 2 projects:
+One project with just pvcs and the other with pvcs attached to a pod
+"""
+import logging
+import time
+import pytest
+from itertools import cycle
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources.ocs import OCS
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-278")
+@pytest.mark.parametrize(
+    argnames=["pvcs_num"],
+    argvalues=[
+        pytest.param(
+            100, marks=pytest.mark.tier1
+        ),
+        pytest.param(
+            1000, marks=pytest.mark.scale
+        )
+    ]
+)
+class TestDeleteProjectWithPVCAndPods(ManageTest):
+    """
+    Create at least 2 projects, one with just pvcs and the other with pvcs
+    attached to pod, and delete them.
+    """
+
+    @pytest.fixture()
+    def setup(self, storageclass_factory):
+        """
+        Create an OCP+OCS Setup, as well as two Storage classes:
+        one for CephFS and one for RBD.
+        Scale the setup to at-least 800 pvcs/pods.
+        """
+        self.ocp_setup = ocp.OCP()
+        self.ocs_setup = OCS()  # TODO: Scale the setup to at-least 800 pvcs/pods
```

Comment:
is the todo comment still relevant?

Reply:
Yes - I will discuss it with you and apply accordingly